pages/api/submit/[collection].ts

```diff
@@ -68,6 +68,7 @@ const submitJob = async (transactionHex: string, signatureHex: string, collectio
   try {
     const validRes = await validateTx(serverAdrrInputValue, outputValueToServerAddr, outputValueToOtherAddr, serverAdrrInputHashes, collection)
     isValid = validRes.isValid
+    console.log("isVald", isValid)
```

Comment:
can you remove console logs? especially from server side, or just do something to log only in debug (it slows down flow a lot, after it's already NOT concurrent for purpose)